.gitignore

```diff
@@ -13,6 +15,7 @@ node_modules
 
 # misc
 .DS_Store
+.env
```

Comment:
you can add .env.* and it will ignore all env files,. .* is a wildcard operator and .env.* will mean add all files .starting with .env. to gitignore